src/main/java/ui/windows/creation/CreationDisplay.java

```diff
@@ -166,14 +166,9 @@ public void actionPerformed(ActionEvent e) {
             }
         }
         boolean gameCreated = controller.createGameResponse(controllerInput);
-        PlayerGameInputBoundary playerGameInteractor = controller.getPlayerGameInteractor();
-        GameBridge bridge = new PlayerGameController(playerGameInteractor);
-        gameController.setBridge(bridge);
         // If a Game was created (valid input for game players), then set the window to the Game.
         if(gameCreated) {
-            gameController.startGame();
-            layoutManager.setPane(WindowName.GAME);
```

Comment:
Im wondering where  layoutmanager.setPane(GAME) will be called, as to switch the panel to GameDIsplay?

Reply:
the new navigation button is linked to a panedelegator thatll handle the switch, this method can be cleaned up but i wasnt sure what to change